presubmit.yml

```diff
@@ -0,0 +1,8 @@
+---
+tasks:
+  ubuntu_1804:
+    platform: ubuntu1804
+    build_targets:
+    - "..."
```

Comment:
Both of these should probably be "//kythe/..." rather than just "..."

Reply:
Also, from the documentation shouldn't this be: `.bazelci/presubmit.yml`?

Reply:
Done